modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -76,7 +76,7 @@ The products you need for this procedure are:
 | SUSE Linux Enterprise Server with Expanded Support 7 x86_64
 
 | {es} 8
-| RHEL or SLES ES or CentOS 8 Base and SUSE Linux Enterprise Server with Expanded Support 8 x86_64
+| RHEL or SLES ES and SUSE Linux Enterprise Server with Expanded Support 8 x86_64
```

Comment:
The channel / product name does not change just because CentOS 8 is now unsupported

Reply:
reverted